deploy/docker/cp-gitlab-reader/init.sh

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 cd  $CP_GITLAB_READER_HOME/gitreader/gitreader
+openssl x509 -pubkey -noout -in $CP_API_SRV_CERT_DIR/jwt.key.x509  > $CP_GITLAB_READER_HOME/pub-jwt-key.pem
```

Comment:
Can't we use `$CP_API_SRV_CERT_DIR/jwt.key.public` instead of generation of a new public key?